layers/+emacs/better-defaults/packages.el

```diff
@@ -10,9 +10,19 @@
 ;;; License: GPLv3
 
 (defconst better-defaults-packages
-  '(mwim)
+  '((ansi-colors :location built-in)
+    mwim)
   "The list of Lisp packages required by the mwim layer.")
 
+(defun better-defaults/init-ansi-colors ()
+  :init
+  (progn
+    (defun better-defaults/compilation-buffer-apply-ansi-colors ()
+      (setq inhibit-read-only t)
+      (ansi-color-apply-on-region compilation-filter-start (point-max))
```

Comment:
**NB:** I used `point-max` here, but it also worked using `point`. I don't know if it is of any importance or if it might impact performance. Insights welcomed.